Makefile

```diff
@@ -3,8 +3,13 @@ LOCALBIN := $(ROOT_DIR)/bin
 
 OUT_DIR := $(ROOT_DIR)/out
 
+GINKGO ?= ginkgo
 GO ?= go
 
+PERF_CLUSTER_PROVIDER ?= kwokctl
+PERF_CLUSTER_KUBECONFIG ?= /tmp/namespace-lister-perf-test
```

Comment:
Maybe we should use `mktemp` or use an `out/` directory that we ignore in git?  I'm not a fan of hardcoded paths like this.

Reply:
Great idea